fission/src/ui/panels/configuring/assembly-config/interfaces/scoring/ZoneConfigInterface.tsx

```diff
@@ -71,7 +71,7 @@ function save(
     const translation = new THREE.Vector3(0, 0, 0)
     const rotation = new THREE.Quaternion(0, 0, 0, 1)
     const scale = new THREE.Vector3(1, 1, 1)
-    gizmo.mesh.matrixWorld.decompose(translation, rotation, scale)
+    gizmo.obj.matrixWorld.decompose(translation, rotation, scale)
```

Comment:
I still don't like that we're doing this from outside the gizmo.

Reply:
At the end of the day, whatever uses the gizmo needs to have the freedom to do whatever it needs with the transform data it provides. I don't think there will ever be a way around this purely.